analysis/measures_definition_pf_codes_conditions.py

```diff
@@ -145,6 +145,24 @@
             intervals=months(monthly_intervals).starting_on(start_date),
         )
 
+denominator_uncomplicated_uti = (age>=16) & (age<=64) & (patients.sex.is_in(["female"]))
```

Comment:
we should add a comment above this line explaining which part of the denominaotr rules we already implemented, and importantly also that we didnt to the exclusions yet.